wremnants/unfolding_tools.py

```diff
@@ -17,7 +17,7 @@ def add_out_of_acceptance(datasets, group, newGroupName=None):
             ds = deepcopy(dataset)
 
             if newGroupName is None:
-                ds.group = "Bkg"+ds.group
+                ds.group = ds.group+"OOA"
```

Comment:
It might be worth discussing also with @emanca . Right now the theory agnostic is no longer implemented with an independent OOA histogram. However, if it ever happens again, some updates to comply with this new scheme for the histogram name might be needed there as well (there are also some parts of setupCombine.py which might probably be ineffective at the moment, but might induce unexpected behaviours if this feature is introduced again)